grammar.js

```diff
@@ -135,14 +140,14 @@ module.exports = grammar({
       '>',
       choice(
         alias(token.immediate(/[a-z0-9]+\n/), $.language),
-        token.immediate('\n')),
+        token.immediate(/\n/)),
```

Comment:
JS regex are for parsing purposes; only use literals if you want to expose them as anonymous nodes to query.

Reply:
regex has lower priority than literals

Reply:
It's not about priority; it's what gets exposed to queries. "Hiding" stuff from queries is the primary way of keeping parser size down (and performance up).

Reply:
> "Hiding" stuff from queries is the primary way of keeping parser size down (and performance up).

Nice. Would be useful to add that tip here: https://github.com/neovim/tree-sitter-vimdoc/blob/ce5ea848f52dfce683884a6ae2d44a0fb637ec2c/grammar.js#L2-L5

Reply:
Tips at home: https://github.com/nvim-treesitter/nvim-treesitter/wiki/Parser-Development

Reply:
> regex has lower priority than literals (oh but that's "only for terminals")

Oh, I see what you mean here. But here it's fine since the `token.immediate` takes care of it. (In fact, it's better to simply omit it from the `language` node since that should _not_ include the newline -- I tested it.)